public/pages/Destinations/containers/CreateDestination/CreateDestination.js

```diff
@@ -49,20 +49,25 @@ class CreateDestination extends React.Component {
 
     const { location, edit, history } = this.props;
     let destinationVersion;
+    let ifSeqNo, ifPrimaryTerm;
     if (edit) {
       // In case user is refreshing in edit mode , redirect them to the destination page.
       // TODO:: Ideally this should fetch the destination from ElasticSearch and fill in value
       const destinationToEdit = _.get(location, 'state.destinationToEdit', null);
       if (destinationToEdit) {
         initialValues = { ...destinationToFormik(destinationToEdit) };
         destinationVersion = destinationToEdit.version;
```

Comment:
Do we need this version anymore?

Reply:
Removed unused version from both the places.